react-function-components/src/index.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+function CustomButton() {
+  return <button>Click Me!</button>;
+}
+const $div = document.getElementById('root');
+const element = CustomButton();
```

Comment:
In React, we don't call our components. Instead, we create a React element.

```jsx
ReactDOM.render(
  <CustomButton />,
  document.querySelector('#root')
);
```

Reply:
oh duhhhh